urllib3/util/connection.py

```diff
@@ -34,7 +37,8 @@ def is_connection_dropped(conn):  # Platform-specific
 # One additional modification is that we avoid binding to IPv6 servers
 # discovered in DNS if the system doesn't have IPv6 functionality.
 def create_connection(address, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
-                      source_address=None, socket_options=None):
+                      source_address=None, socket_options=None,
+                      enable_happy_eyeballs=True):
```

Comment:
Good catch! This was added a long time ago before I realized that create_connection is buried really deep in the API, I wanted to give a way for users to disable the algorithm if they need to but I don't think it's worth it to expose it at a high level. I'll remove the param. :)